core/mempool/src/adapter/mod.rs

```diff
@@ -639,7 +639,7 @@ mod tests {
             M: MessageCodec,
             P: AsRef<[Bytes]> + Send + 'a,
         {
-            unreachable!()
+            Ok(())
```

Comment:
Looks like this is the only non-trivial change.

Reply:
This is required by `clippy`.

Reply:
Shall we add comment to make `clippy` silence here, avoid changing any Axon logic in this PR?

Reply:
These code is in unit test.